include/util/packed_vector.hpp

```diff
@@ -287,6 +313,12 @@ template <typename T, std::size_t Bits, storage::Ownership Ownership> class Pack
             return &container == &other.container && internal_index == other.internal_index;
         }
 
+        // FIXME: This is needed for tests on Boost ranges to correctly compare Alias values.
+        template <typename F, typename U> bool operator!=(const osrm::Alias<F, U> value) const
```

Comment:
This is a bit of a hack. The unit tests compare PackedVector ranges of Alias types against normal vectors of Alias types. Boost seems to know what to do for integral types, but this was the only way to get the comparison to work. Open to suggestion of how to do this in a better way.